From 602fc626d157e66d0436f99f8d8df30f2022182b Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Fri, 29 Oct 2021 20:17:42 +0200
Subject: [PATCH 2/2] Replace usages of FULL paths with their relative
 counterparts

GNUInstallDirs currently doesn't support the seperate STAGING directory,
and will always prepend the final installation prefix to its paths.

Work around that by just using the path that is relative to the
installation directory and let CMake figure out the rest.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e744d17..b574e6f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2630,7 +2630,7 @@ if(NOT (WINDOWS OR CYGWIN))
   endif()
   install(PROGRAMS ${SDL2_BINARY_DIR}/sdl2-config DESTINATION bin)
   # TODO: what about the .spec file? Is it only needed for RPM creation?
-  install(FILES "${SDL2_SOURCE_DIR}/sdl2.m4" DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/aclocal")
+  install(FILES "${SDL2_SOURCE_DIR}/sdl2.m4" DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/aclocal")
 endif()
 
 ##### Uninstall target #####
-- 
2.36.1

